app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -120,6 +122,18 @@ def sp_or_app_name
       current_sp&.friendly_name.presence || APP_NAME
     end
 
+    def upload_disabled?
+      if doc_auth_vendor == Idp::Constants::Vendors::SOCURE
+        return true unless desktop_test_mode_enabled?
+      end
+
+      idv_session.selfie_check_required && !desktop_test_mode_enabled?
+    end
+
+    def desktop_test_mode_enabled?
+      idv_session.desktop_selfie_test_mode_enabled? || idv_session.desktop_socure_test_mode_enabled?
```

Comment:
```suggestion
      !idv_session.desktop_selfie_test_mode_enabled? &&
        (doc_auth_vendor == Idp::Constants::Vendors::SOCURE || idv_session.selfie_check_required )
```
it looks like we forgot to Cancel [LG-14809](https://cm-jira.usa.gov/browse/LG-14809) after our refinement session last week in which we concluded to reuse the selfie desktop flag here and then rename it later in [LG-14925](https://cm-jira.usa.gov/browse/LG-14925).  I suppose you could proceed as is, but we really do not need to add anew flag. We could just go ahead and remove the desktop_socure_test_mode_enabled? here